wetransfer_style.gemspec

```diff
@@ -1,15 +1,22 @@
 Gem::Specification.new do |s|
   s.name = "wetransfer_style"
-  s.version = "0.6.2"
+  s.version = "0.6.4"
   s.summary = "At WeTransfer we code in style. This is our style."
   s.description = s.summary
   s.homepage = "https://github.com/WeTransfer/wetransfer_style"
   s.license = "MIT"
 
   s.files = Dir["README.md", "LICENSE", "ruby/*.yml"]
 
-  s.add_dependency "rubocop", "0.52.1"
+  # lets not allow a major bump for Rubocop, but there is nothing special about 0.61.1
```

Comment:
This is major and minor, just to make sure.

Not sure how rubocop does updates, but we might stick to a fixed version, since any _slight_ change can break the projects' style that use this gem.

Reply:
Sorry. `~> 0.61.1` resolves to `0.61.x`, where `x >= 1`. So, only the patch version is changed.

Reply:
Thanks for the clarification. The `~> 0.61.1` was intended as `~> 0.61`. Good catch. 

But reading their [installation notes](https://github.com/rubocop-hq/rubocop#installation) seems to resolve this discussion: 

> RuboCop's development is moving at a very rapid pace and there are often backward-incompatible changes between minor releases (since we haven't reached version 1.0 yet). To prevent an unwanted RuboCop update you might want to use a conservative version locking in your `Gemfile`:
> 
> `gem 'rubocop', '~> 0.62.0', require: false`

Would leaving the version specification like this resolve your tensions?

Reply:
I considered it was better to be safe than sorry, that's why I went for the hard version; you never know how gem authors bump versions. I also had a discussion with @julik and we settled on this. But I'm not against this change; we can try it and if things don't break unexpectedly, it's fine 👍

Reply:
> I think it would make sense to pin to a specific version in the repo that uses this gem.

Nope. See below.